helm/templates/deployment.yaml

```diff
@@ -48,6 +48,9 @@ spec:
             - name: {{ .Chart.Name }}-certs
               mountPath: /etc/certs
               readOnly: true
+            - name: {{ .Chart.Name }}-alertconfig
+              mountPath: "/app/config"
```

Comment:
Now, we have two places where we hard-coded the alert configuration directory.

@phbelitz @peterthomassen Is there a better way? I thought about handing it over via command line as flag for `helm install` but that seemed like unnecessary overhead to me :-/

Reply:
i don't see any major problems with that. i don't expect we'll be chnaging the directory any time soon.

Reply:
I have no opinion as I don't know my way around helm.

Reply:
ok, thanks - I will resolve it then :)